toobusy.cc

```diff
@@ -85,13 +85,22 @@ static void every_second(uv_timer_t* handle, int status)
     s_lastMark = now;
 };
 
-extern "C" void init(Handle<Object> target) {
-    HandleScope scope;
+static void every_second(uv_timer_t* handle, int) {
```

Comment:
If you put `#pragma GCC diagnostic ignored "-Wunused"` on the line before this, it will suppress the clang and GCC `unused function` warnings on compile.